grails-app/views/project/_details.gsp

```diff
@@ -1,434 +1,506 @@
+            <div class="clearfix control-group">
+                <label class="control-label span3" for="plannedStartDate"><g:message code="project.details.plannedStartDate"/>
+                <fc:iconHelp><g:message code="project.details.plannedStartDate.help"/></fc:iconHelp><i class="req-field"></i></label>
 
-            <div class="controls span9">
-                <fc:datePicker class="input-small" targetField="plannedStartDate.date" name="plannedStartDate"
-                               id="plannedStartDate" data-validation-engine="validate[required]"/>
+                <div class="controls span9">
+                    <fc:datePicker class="input-small" targetField="plannedStartDate.date" name="plannedStartDate"
+                                   id="plannedStartDate" data-validation-engine="validate[required]"/>
+                </div>
             </div>
-        </div>
 
-        <div class="clearfix control-group">
-            <label class="control-label span3" for="plannedEndDate"><g:message code="project.details.plannedEndDate"/>
-            <fc:iconHelp><g:message code="project.details.plannedEndDate.help"/></fc:iconHelp>
-            </label>
+            <div class="clearfix control-group">
+                <label class="control-label span3" for="plannedEndDate"><g:message code="project.details.plannedEndDate"/>
+                <fc:iconHelp><g:message code="project.details.plannedEndDate.help"/></fc:iconHelp>
+                </label>
+
+                <div class="controls span9">
+                    <fc:datePicker class="input-small" targetField="plannedEndDate.date" name="plannedEndDate"
+                                   id="plannedEndDate" data-validation-engine="validate[future[plannedStartDate]]"/>
+                    <g:message code="project.details.plannedEndDate.extra"/>
+                </div>
+            </div>
+
+            <div id="associatedOrgs">
+                <div class="row-fluid">
+                    <div class="clearfix control-group">
+                        <label class="control-label span3" for="associatedOrgList"><g:message code="project.details.associatedOrgs"/>:<fc:iconHelp><g:message code="project.details.associatedOrgs.help"/></fc:iconHelp></label>
+                        <div class="span6" id="associatedOrgList">
+                            <!-- ko foreach: associatedOrgs -->
+                            <div class="span12 margin-left-0 margin-bottom-1">
+                                <div class="span5 margin-left-0" data-bind="text: name"></div>
+                                <div class="span5"><img src="" data-bind="visible: logo, attr: {src: logo}" alt="Organisation logo" class="small-logo"></div>
+                                <div class="span2"><a href="#" data-bind="click: $parent.removeAssociatedOrganisation, attr: {'data-value': id}"><i class="fa fa-remove">&nbsp;</i><g:message code="project.details.associatedOrgs.remove"/></a></div>
```

Comment:
for consistency with rest of the page, it might be better to have this as a button.